### PR TITLE
Update 10 minutes to cuDF and CuPy with new APIs

### LINKED ISSUE #7160: [DOC] Not 100% accurate statement in 10 min to cuDF and CuPy
## Report incorrect documentation
Hi!

I think the following paragraph is not 100% accurate:

> Because CuPy arrays have a single dtype, each column in our DataFrame must have the same dtype, **regardless of which method we use.**

The following code works just fine:

```
import cudf
import cupy

df = cudf.DataFrame({'a': 3, 'b': 3.0})
arr = cupy.asarray(df.as_gpu_matrix())
```

In the example above, `int` datatypes as casted to `float64`.

Hope it helps!
Miguel

**Location of incorrect documentation**
https://github.com/rapidsai/docs/blob/gh-pages/api/cudf/0.18/10min-cudf-cupy.ipynb

### PR DESCRIPTION
**Pull request by ChrisJar**
This updates the 10 minutes to cuDF and CuPY notebook to use the new methods for moving between cuDF data structures and CuPy arrays.

Closes #7160

### CODE REVIEW

File: docs/cudf/source/10min-cudf-cupy.ipynb [hunk 1]
```diff
@@ -17,10 +17,8 @@
    "source": [
```
Comment (shwina) [inline docs/cudf/source/10min-cudf-cupy.ipynb:17]: <p>Given these timings, why are we recommending (1) over (3)? It looks like it's about 8 times slower. Also are there any differences between <code>%time</code>  and <code>%timeit</code>  results here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/7158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (shwina) [inline docs/cudf/source/10min-cudf-cupy.ipynb:17]: <p>Ditto here, should we be recommending the fastest approach instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/7158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (beckernick) [reply docs/cudf/source/10min-cudf-cupy.ipynb:17]: <p>Is this difference just showing the JIT overhead, or is <code>.values </code> slower than <code>as_gpu_matrix</code>?</p>

Comment (ChrisJar) [reply docs/cudf/source/10min-cudf-cupy.ipynb:17]: With `%timeit` the differences are less extreme, but `.values` is still the slowest for both `DataFrame` and `Series`.

**Review by shwina — APPROVED** [review]
LGTM!

**Comment by ChrisJar** [comment]
rerun tests

**Comment by kkraus14** [comment]
@gpucibot merge

**Pull request merged**